File: JUnit Testing/JUnit/src/Calc/Calculator.java
package Calc;

public class Calculator {
	 public int add(int a, int b) { 
	        return a + b; 
	    } 
	 
	 
	    public int subtract(int a, int b) { 
	        return a - b; 
	    } 
	 
	    
	    public int multiply(int a, int b) { 
	        return a * b; 
	    } 
	 
	    
	    public int divide(int a, int b) { 
	        if (b == 0) { 
	            throw new IllegalArgumentException("Cannot divide by zero."); 
	        } 
	        return a / b; 
	    }
	    
	    
	    public int power(int base, int exp) {
	    	int result = 1;
	    	
	    	for(int i=0; i<exp; i++) {
	    		result = result * base;
	    	}
	    	
	    	return result;
	    }
	    
	    
	    public int modulus(int a, int b) {
	    	if(b == 0) {
	    		throw new IllegalArgumentException("Cannot modulus by zero.");
	    	}
	    	
	    	return a % b;
	    }

}

File: Pytest Unit Testing/Home Work/calculator.py
class Calculator:

    def add(self, a, b):
        return a + b

    def subtract(self, a, b):
        return a - b

    def multiply(self, a, b):
        return a * b

    def divide(self, a, b):
        if (b==0):
            raise ValueError("Can't divide by zero")
        return a / b

c = Calculator()
print(c.add(5,2))
print(c.subtract(5,2))
print(c.multiply(5,2))
print(c.divide(5,2))